src/components/posters/Posters.js

```diff
@@ -3,23 +3,33 @@ import Poster from '../poster/Poster';
 import './Posters.css';
 
 const Posters = ({movies, selectMovie}) => {
-  const posterCards = movies.map(movie => {
+  let placeholderIDs = [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 ]
+  const placeholderPosterCards = placeholderIDs.map(number => {
     return (
-      <Poster
-        key={movie.id}
-        id={movie.id}
-        photo={movie['poster_path']}
-        title={movie.title}
-        selectMovie={selectMovie}
-      />
+      <Poster key={number}/>
```

Comment:
Could you explain why this is necessary? Usually, we'd just use the ID from the movie, as the key _and _ the ID